### DISCUSSION TITLE
unable to login into chrome profile

### DISCUSSION THREAD

**Discussion by 810D** [ultrafunkamsterdam/undetected-chromedriver#1204] [Q&A]
Hello,

i used to use undected_chromedriver for a project a while ago, and everyhing used to work. (version 3.0.3). Today, I wanted to use it again, but I ran into some issues. The newer version (3.4.6) does not want to login to my chrome profile. I tried it mutiple different ways on multiple versions.

**Way 1:**
    chrome_options = uc.ChromeOptions()
    chrome_options.add_argument(r'--user-data-dir=C:\Users\name\AppData\Local\Google\Chrome\User Data')
    chrome_options.add_argument(r'--profile-directory=Profile 3')
    driver = uc.Chrome(chrome_options = chrome_options)

Problem(3.4.6): Does not log into profile.
Problem(3.0.3) Error after succesfully logging in :   raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited normally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location C:\Program Files\Google\Chrome\Application\chrome.exe is no longer running, so ChromeDriver is assuming that Chrome has crashed.)

**Way 2:** 
    options = uc.ChromeOptions()
    #chrome_options.add_argument(r'--user-data-dir=C:\Users\danie\AppData\Local\Google\Chrome\User Data')
    #chrome_options.add_argument(r'--profile-directory=Profile 3')
    options.user_data_dir = "C:\Users\name\AppData\Local\Google\Chrome\User Data\Profile 3"
    driver = uc.Chrome(chrome_options = options)

Problem: Not logging in.

I tired some other ways too, including the way it is in the documentation, but it's not working.
It would be good to be able to use it, and my chrome version might be the problem, as that is the only thing consistent in these. It is version  112.0.5615.121.

I'd be very thankful for some help, and you sharing your chrome version, udc version and working code with me.

**Comment by TheGreatSimo** [comment]
Same problem here
